docs/design/server_side_charts.md

```diff
@@ -0,0 +1,44 @@
+# Supporting charts on the server side
+As discussed in #367, the server side components manager and resourcifier have not yet been refactored, as of this writing, to work with [the new chart format](docs/design/chart_format.md). Presumably, expandybird should not be affected.
+
+## Context
+In discussions about the refactoring on the Slack channel, a number of potential requirements were discussed that could complicate matters, such as:
+* supporting multiple templating technologies within a single chart.
+* supporting multiple templating technologies within a single dependency graph.
```

Comment:
Re: letting each templating engine provide its own dependency resolution mechanism, what will they depend on (if not charts) and where will they fetch them from?

Maybe we leave in the old DM github integration for this purpose, with the intention of deprecating it later?

In the simple case, they can import other files from the same dir, but we get that for free and it's not really what we're talking about here.